includes/definitions/quantastor.yaml

```diff
@@ -0,0 +1,43 @@
+os: quantastor
+text: 'QuantaStor'
+type: storage
+icon: quantastor
+ifXmcbc: true
+ifname: true
+mib_dir: osnexus
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery_modules:
```

Comment:
Please don't disable modules at the OS level, especially ones that are disabled by default.